core/embed/rust/src/ui/layout_delizia/ui_firmware.rs

```diff
@@ -126,6 +126,7 @@ impl FirmwareUI for UIDelizia {
         _warning_footer: Option<TString<'static>>,
         external_menu: bool,
     ) -> Result<Gc<LayoutObj>, Error> {
+        debug_assert!(!(info && external_menu));
```

Comment:
We're still not running tests with `debug_assert!`s: https://github.com/trezor/trezor-firmware/pull/6033#issuecomment-3433139903
Can we make return `Error::NotImplementedError` here?